conviva/src/main/java/com/bitmovin/analytics/conviva/ssai/DefaultPlaybackInfoProvider.java

```diff
@@ -33,6 +33,10 @@ public HashMap<String, Object[]> getPlaybackVideoData() {
         if (playbackVideoData != null) {
             videoData.put(ConvivaSdkConstants.PLAYBACK.RESOLUTION, new Object[]{playbackVideoData.getWidth(), playbackVideoData.getHeight()});
             videoData.put(ConvivaSdkConstants.PLAYBACK.BITRATE, new Object[]{playbackVideoData.getBitrate() / 1000});
+            int averageBitrate = playbackVideoData.getAverageBitrate();
+            if (averageBitrate != VideoQuality.BITRATE_NO_VALUE) {
+                videoData.put(ConvivaSdkConstants.PLAYBACK.AVG_BITRATE, new Object[]{averageBitrate / 1000});
+            }
```

Comment:
I think we should only report `peakBitrate` to `ConvivaSdkConstants.PLAYBACK.BITRATE`, not potentially the `averageBitrate` if  `peakBitrate` is not filled.

Reply:
good catch :+1: 
fixed with [886ee97](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/77/commits/886ee973f9be49c289d879f3d50e5fb8df1aa155)